frontend/src/components/EditRecipe/EditRecipe.js

```diff
@@ -0,0 +1,214 @@
+                  <div className='ingredient-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='ingredient-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='ingredient-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='ingredient-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='ingredient-input'><input className='form-control' type='text' placeholder=''></input></div> 
+                </div>
+              </div>
+              <div className='col-1'>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='row add-button'>
+          <div className='col-2 offset-7'>
+            <button type='button' className='btn btn-primary btn-lg'><i class="fas fa-plus"></i>ADD</button>
+          </div>
+        </div>
+        <div className='row directions-title'>
+          <div className='col-8 offset-2'>
+            <div>Directions</div>
+          </div>
+        </div>   
+        <div className='row'>
+          <div className='col-8 offset-2'>
+            <div className='row form-directions'>
+              <div className='col-7 offset-2'>
+                <div className='directions form-group'>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div>
+                  <div className='direction-input'><input className='form-control' type='text' placeholder=''></input></div> 
+                </div>
+              </div>
+              <div className='col-1'>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
+                <div className='row delete-input'><i class="fas fa-times"></i></div>
```

Comment:
I don't want to be a pixel-perfect lunatic, but I think these could be a bit bigger.